multi-node/aws/pkg/config/templates/cloud-config-worker

```diff
@@ -59,6 +59,29 @@ coreos:
         [Install]
         RequiredBy=kubelet.service
 
+    {{ if .Experimental.NodeDrainer.Enabled }}
+    - name: kube-node-drainer.service
+      enable: true
+      content: |
+        [Unit]
+        Description=drain this k8s node to make running pods time to gracefully shut down before stopping kubelet
+        After=kubelet.service
+
+        [Service]
+        Type=oneshot
+        RemainAfterExit=true
+        ExecStart=/bin/true
+        ExecStop=/bin/sh -c '/usr/bin/docker run --rm -v /etc/kubernetes:/etc/kubernetes {{.Experimental.NodeDrainer.KubectlImage}} \
+          {{.Experimental.NodeDrainer.KubectlCommand}} \
+          --server=https://{{.ControllerIP}}:443 \
+          --kubeconfig=/etc/kubernetes/worker-kubeconfig.yaml \
+          drain $$(hostname) \
+          --force'
+
```

Comment:
Does this also need --ignore-daemonsets=true?  The man page implies to me that you'd need both if you had a daemonset and an unmanaged pod, even though the name "force" makes me think "force" would be enough.